test/pubsub.spec.js

```diff
@@ -515,19 +515,15 @@ describe('publish/subscribe', function () {
                         sub.stream.once('data', function () {
                             assert.strictEqual(sub.message_buffers, false);
                             assert.strictEqual(sub.shouldBuffer, false);
-                            sub.on('pmessageBuffer', function (pattern, channel, message) {
-                                if (parser !== 'javascript' && typeof pattern === 'string') {
+                            sub.on('pmessageBuffer', function (pattern, channel) {
+                                if (typeof pattern === 'string') {
                                     pattern = new Buffer(pattern);
                                     channel = new Buffer(channel);
                                 }
                                 assert.strictEqual(pattern.inspect(), new Buffer('*').inspect());
                                 assert.strictEqual(channel.inspect(), new Buffer('/foo').inspect());
                                 sub.quit(end);
                             });
-                            if (parser === 'javascript') {
-                                assert.notStrictEqual(sub.message_buffers, sub.buffers);
```

Comment:
Should this assertion not be kept?

Reply:
Good catch, would have given my code a review before merging don't worry 😅 

I've put this back locally however it causes a failure;

![image](https://user-images.githubusercontent.com/5347038/73982617-7a738080-492c-11ea-9390-f6a24a40c908.png)


This line though doesn't make sense as to what it's testing, do you have any idea what it's trying to test?

Reply:
Figured out the issue